tools/experiments/experiments.js

```diff
@@ -346,6 +346,10 @@ const EXPERIMENTS = [
     name: 'Enables scroll snap on carousel on Chrome browsers',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/16508',
   },
+  {
+    id: 'amp-viewer-assistance-untrusted',
+    name: 'Enable amp-viewer-assistance in untrusted viewers',
+  },
```

Comment:
Can we remove this? It's a bit confusing for a local-testing trigger to be listed with other experiments. Or do you need this for local testing on device?

Reply:
Yes, removed.